packages/rum-core/test/performance-monitoring/capture-navigation.spec.js

```diff
@@ -316,4 +316,24 @@ describe('Capture hard navigation', function() {
     expect(tr.marks.agent.firstContentfulPaint).toBeUndefined()
     unMock()
   })
+
+  it('should not add API calls as resource timing spans', function() {
+    const unMock = mockGetEntriesByType()
+    const tr = new Transaction('test', PAGE_LOAD, {
+      startTime: transactionStart
+    })
+    const apiSpan = tr.startSpan('GET http://ajax-filter.test', 'external.http')
+    apiSpan.end()
+    tr.captureTimings = true
+    tr.end(transactionEnd)
+    captureNavigation(tr)
+
+    const filteredRTSpan = tr.spans.filter(
+      span => span.name === 'http://ajax-filter.test'
```

Comment:
Changing this check into something like `span.name.indexOf('http://ajax-filter.test')>-1` would be better, e.g. if there are changes to the resource timing span names that would affect this but might not necessarily break this test.

Reply:
we have to test also the type in that case as it would match the API span name